units/commit_queue.go

```diff
@@ -187,12 +198,20 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 		}
 
 		grip.Info(message.Fields{
-			"source":  "commit queue",
-			"job_id":  j.ID(),
-			"item":    nextItem,
-			"message": "finished processing item",
+			"source":               "commit queue",
+			"job_id":               j.ID(),
+			"item":                 nextItem,
+			"message":              "finished processing commit queue item",
+			"processing_time_secs": time.Since(beginProcessingTime).Seconds(),
```

Comment:
I don't really expect processing time to be a bottleneck; i'm not sure this log has value